f8a_worker/data_normalizer.py

```diff
@@ -227,8 +226,12 @@ def _rf(iterable):
                         base['engines'][name] = operator_version
         elif isinstance(engines, str):
             # 'node 4.2.3' -> {"node": "4.2.3"}
-            name, version = engines.split()
-            base['engines'] = {name: version}
+            try:
+                name, version = engines.split()
+                base['engines'] = {name: version}
+            except Exception:
```

Comment:
We can be more specific here, i.e. we can catch `ValueError` exceptions.

Reply:
Also I'd add comment here that we are giving up as the data is malformed.

Reply:
Or just change `elif isinstance(engines, str):` to `elif isinstance(engines, str) and ' ' in engines:`.
Basically all the `_handle_javascript()` code is handling malformed data since nodejs package maintainers don't care about https://docs.npmjs.com/files/package.json and you'd soon end up with lots of `try` `expect`.

Reply:
And that's why Node developers have no friends! :)

`elif isinstance(engines, str) and ' ' in engines:` would still throw an exception if there is more values to unpack. I'd maybe keep this simple and just catch the exception here. Exception means the data is weird and we don't care, we ignore it.

@jpopelka glad to see you here! :)